src/modules.js

```diff
@@ -110,7 +110,8 @@ var LibraryManager = {
         'library_nodefs.js',
         'library_sockfs.js',
         'library_workerfs.js',
-        'library_tty.js'
+        'library_tty.js',
+        'library_lz4.js',
```

Comment:
:(

Perhaps we should start gating on the linker flag `-llz4.js` or `--library_js lz4.js` or something similar for  new features here, and not pull them in by default?